MAINTAINERS.md

```diff
@@ -0,0 +1,14 @@
+# Kubernetes Event-driven Autoscaling (KEDA) Maintainers
+
+## Maintainers
+
+| Maintainer           | GitHub ID                                     | Affiliation |
+| -------------------- | --------------------------------------------- | ----------- |
+| Jeff Hollan          | [jeffhollan](https://github.com/jeffhollan)   | Microsoft   |
+| Anirudh Garg         | [anirudhgarg](https://github.com/anirudhgarg) | Microsoft   |
+| Aarthi Saravanakumar | [Aarthisk](https://github.com/Aarthisk)       | Microsoft   |
+| Yaron Schneider      | [yaron2](https://github.com/yaron2)           | Microsoft   |
```

Comment:
```suggestion
| Satish Ranjan | [SatishRanjan](https://github.com/SatishRanjan)       | Microsoft   |
```

Reply:
I would ask to not be removed.

Reply:
Yeah per earlier conversations for this specific markdown we are going to try to keep to just the active maintainers.  I think it does make sense to give credit to initial ones, potentially even another section in this?  FWIW in the CNCF proposal I don't plan on changing that.  This would be more a reference for folks needing insight in current day to day of KEDA. https://github.com/cncf/toc/pull/383

Reply:
That makes sense to me, does that sound fair to you @yaron2 @Aarthisk ?

Reply:
yep

Reply:
Introduced alumni and added @yaron2 & @Aarthisk to that but will wait for @Aarthisk  to agree on that.